### PR TITLE
Add "Feature section" component page

### PR DESCRIPTION
**Pull request by shixiedesign**
### Related Ticket(s)

#1033

### Description

New component, new page!

### Changelog

**New**

- Add "Feature section" page

### CODE REVIEW

**Comment by shixiedesign** [comment]
Here's a new page added @janchild . It's intentionally pretty barebone and only to keep the site up-to-date with all the links. We can fill in more content & images of use cases later.

https://ibmdotcom-library.s3-web.us-south.cloud-object-storage.appdomain.cloud/deploy-previews/1086/components/feature-section/

File: src/pages/components/feature-section.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<AnchorLink>Design and functional specifications</AnchorLink>
+<AnchorLink>Development documentation</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+
+## Default
+
+Feature section is a full-width section, designed to create a visual break from the rest of the page and focus the viewer's attention to one single story. It contains expressive imagery and a CTA linking people to more details.
```
Comment (janchild) [inline src/pages/components/feature-section.mdx:29]:
```suggestion
Feature section is a full-width section, designed to create a visual break from the rest of the page and focus the viewer's attention to one single story. It contains expressive imagery and a CTA linking people to more details.
```

**Review by janchild — COMMENTED** [review]
Made a couple of copy editing tweaks. Waiting for next build to check. @shixiedesign looks like the build didn't automatically kick off. Not sure why not.

File: src/pages/components/feature-section.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+import ComponentDescription from "components/ComponentDescription";
+import ComponentFooter from "components/ComponentFooter";
+
+<ComponentDescription name="Feature section" type="layout" />
+
+<InlineNotification>
+
+**Note:** Additional usage guidelines are coming soon.
```
Comment (janchild) [inline src/pages/components/feature-section.mdx:13]:
```suggestion
Additional usage guidelines will be available soon.
```

**Review by janchild — COMMENTED** [review]
@shixiedesign Adding this for you to commit to see if it will kick off a new build of the changes.

**Pull request merged**